app/assets/javascripts/controllers/SearchCtrl.coffee

```diff
@@ -0,0 +1,29 @@
+@bruseApp.controller 'SearchCtrl', ['FileHandler', '$scope', '$http', (FileHandler, $scope, $http) ->
+  $scope.files = []
+
+  # watches the search input field for changes
+  $scope.$watch "search", () ->
```

Comment:
Maby it would be nice to send old and new values into the watch callback? That way we can compare them and avoid unnecessary request to the server?

Like this:

```coffee
$scope.$watch('search', (newValue, oldValue) ->
  unless newValue == oldValue
    # do things...
```

Also, [avoid empty parameter lists](https://github.com/styleguide/javascript).